### PR TITLE
Fix win install message

### PR DESCRIPTION
**Pull request by flotwig**
- Closes 

### User facing changelog

n/a?

### Additional details

- windows urls are broken, they point to the zip and not the tgz and don't work: https://github.com/cypress-io/cypress/commit/a74bdff1a866fb27c618afee8c01cb81a4b635b7
- 
image

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue been tagged with a release in ZenHub? 
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](cli/types/index.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](cli/schema/cypress.schema.json)?

### CODE REVIEW

**Comment by bahmutov** [comment]
@flotwig - does Windows understand `export FOO=...` syntax? I thought for Windows we need to use `set FOO=...`

Screen Shot 2020-01-27 at 11 49 50 AM

**Comment by flotwig** [comment]
> @flotwig - does Windows understand `export FOO=...` syntax? I thought for Windows we need to use `set FOO=...`
> 
> Screen Shot 2020-01-27 at 11 49 50 AM

oops, yeah. but the `set` command doesn't work in all scenarios... i prefer telling windows users to use `cross-env`, since it will work in all shells, what do you think?

```
npm i -g cross-env
cross-env CYPRESS_INSTALL_BINARY=... npm i -g ...
```

**Comment by bahmutov** [comment]
when doesn't `set FOO=...` work, I really would prefer plain environment variable syntax, rather than cross-env

**Comment by flotwig** [comment]
@bahmutov i'm glad you asked :smile: 

`set` doesn't work in powershell:

image

`set` doesn't work in git bash, since it actually invokes the `set` shell builtin:

image

`set` actually doesn't even appear to work in cmd.exe:

image


and `setx` actually sets the variable permanently for all future sessions, so that is undesirable.

**Comment by flotwig** [comment]
Example comment:

---

Unknown CI has built the `win32 x64` version of the Test Runner.

You can install this pre-release platform-specific build using instructions at [https://on.cypress.io/installing-cypress#Install-pre-release-version](https://on.cypress.io/installing-cypress#Install-pre-release-version).

You will need to use custom `CYPRESS_INSTALL_BINARY` url and install Cypress using an url instead of the version.

Instructions are included below, depending on the shell you are using.

#### In Command Prompt (`cmd.exe`):

    set CYPRESS_INSTALL_BINARY=bar
    npm install foo

#### In PowerShell:

    $env:CYPRESS_INSTALL_BINARY = bar
    npm install foo

#### In Git Bash:

    export CYPRESS_INSTALL_BINARY=bar
    npm install foo

#### Using `cross-env`:

If the above commands do not work for you, you can also try using `cross-env`:

    npm i -g cross-env
    cross-env CYPRESS_INSTALL_BINARY=bar npm install foo

File: appveyor.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 branches:
   only:
     - master
-    - develop
+    - fix-win-install-message
```
Comment (bahmutov) [inline appveyor.yml]: oops, just noticed this, please remove branch, add back `develop`

Comment (flotwig) [reply appveyor.yml]: yup will do once i i see cypress-bot post the right thing

**Pull request merged**